common/src/main/java/io/pravega/common/cluster/ClusterException.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.common.cluster;
+
+public class ClusterException extends RuntimeException {
```

Comment:
Here is my reasoning to make this runtime, you let me know if it makes sense.

We are throwing this exception because of an error while dealing with the component that backs the cluster implementation, in our case zookeeper. I'd expect the implementation internally to retry whenever possible and only throw when not possible. If this is right, then there isn't really anything the application can do.

Also, it doesn't necessarily justify, but I'm following what we adopted for `StoreException` in the controller. If we change it here, then perhaps we should change out there too.